src/lib/createAnimationStyles.tsx

```diff
@@ -17,12 +17,18 @@ const createAnimationStyles = (animation: AnimationProp, theme: ThemePrepared) =
         iterationCount,
         playState,
         timingFunction,
+        keyframeParams,
       } = animations[animationName]
 
+      const mergedKeyframeParams =
```

Comment:
sorry, not sure that I completely understand that - shouldn't we actually merge the values that were provided by client with the ones that are defined as default ones for the animation object? I might expect this being useful when, for example, there are multiple params that define start and end position of animated element, and client might want to override just one of them (say, just `startPosition`)

Reply:
Valid point, will add this..